kpatch-build/create-diff-object.c

```diff
@@ -252,129 +252,80 @@ static int kpatch_mangled_strcmp(char *s1, char *s2)
 
 static int rela_equal(struct rela *rela1, struct rela *rela2)
 {
-#ifdef __powerpc__
-	struct section *toc_relasec1, *toc_relasec2;
-	struct rela *r_toc_relasec1, *r_toc_relasec2;
+	struct rela *r_rela1, *r_rela2;
```

Comment:
Let's call them rela_toc1 and rela_toc1 for consistency with the rest of the code.